UnitTests/PayPalUI_Tests/PayPalPayLaterButton_Tests.swift

```diff
@@ -0,0 +1,32 @@
+import XCTest
+import SwiftUI
+import Foundation
+@testable import PayPalUI
+
+class PayPalPayLaterButton_Tests: XCTestCase {
+    
+    func testInit_whenPayPalPayLaterButtonCreated_hasDefaultUIValues() {
+        let sut = PayPalPayLaterButton()
+        XCTAssertEqual(sut.edges, PaymentButtonEdges.softEdges)
+        XCTAssertEqual(sut.size, PaymentButtonSize.collapsed)
+        XCTAssertEqual(sut.color, PaymentButtonColor.gold)
+        XCTAssertEqual(sut.label, .payLater)
+        XCTAssertNil(sut.insets)
+    }
+
+    func testMakeCoordinator_whenPayPalPayLaterButtonRepresentableIsCreated_actionIsSetInCoordinator() {
+        let expectation = expectation(description: "Action is called")
+        let sut = PayPalPayLaterButton.Representable {
+            expectation.fulfill()
+        }
+        let coordinator = sut.makeCoordinator()
+
+        coordinator.onAction(self)
+        XCTAssertNotNil(sut)
+        waitForExpectations(timeout: 1) {error in
+            if let error = error {
+                XCTFail("waitForExpectationsWithTimeout errored: \(error)")
+            }
+        }
+    }
```

Comment:
This is a really neat!

Originally I was thinking we could do something like:
```
func testSample() {
    let fakeAction: () -> Void = { print("Fake action") }
    let sut = PayPalPayLaterButton.Representable(fakeAction)
    let coordinator = sut.makeCoordinator()
        
    XCTAssertTrue(coordinator.action === fakeAction)
}
```
But this isn't possible since [Swift closures aren't equatable](https://www.jessesquires.com/blog/2021/04/05/why-swift-closures-are-not-equatable/).

Reply:
I really like your approach with this test! Some minor fixups to make the test easier to read, including renaming it a bit to be more clear that we're testing using `onAction`:

```suggestion
      // MARK: - PayPalPayLaterButton.Representable for SwiftUI

      func testMakeCoordinator_whenOnActionIsCalled_executesActionPassedInInitializer() {
        let expectation = expectation(description: "Action is called")
        let sut = PayPalPayLaterButton.Representable {
            expectation.fulfill()
        }
        let coordinator = sut.makeCoordinator()

        coordinator.onAction(self)
        waitForExpectations(timeout: 1) {error in
            if error != nil {
                XCTFail("Action passed in PayPalPayLaterButton.Representable was never called.")
            }
        }
    }
```